kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -10,15 +10,15 @@ These scripts can be helpful when scripting the life cycle of a WebLogic Server
 The `startServer.sh` script starts a WebLogic Server in a domain. For clustered Managed Servers, either it increases the `spec.clusters[<cluster-name>].replicas` value for the Managed Server's cluster by `1` or updates the `spec.managedServers[<server-name>].serverStartPolicy` attribute of the domain resource or both as necessary. For the Administration Server, it updates the value of the `spec.adminServer.serverStartPolicy` attribute of the domain resource. For non-clustered Managed Servers, it updates the `spec.managedServers[<server-name>].serverStartPolicy` attribute of the domain resource. The script provides an option to keep the `spec.clusters[<cluster-name>].replicas` value constant for clustered servers. See the script `usage` information by using the `-h` option.
 
 Use the following command to start the server either by increasing the replica count or by updating the server start policy:
-```shell
+```
```

Comment:
Why did you remove "shell"?

Reply:
Tom mentioned that the "shell" view was confusingly highlighting keywords like 'for' and '!' in the 'INFO' lines. I didn't see much difference in the formatting when not using "shell".  Do you see any issue or difference in formatting due to this change? Thanks.

Reply:
I had just gone through and added `shell` because it seemed to do better formatting than `bash`. I thought it was helpful :).

Reply:
Okay, we can leave this alone now, but I'll probably come through later and restore the `shell`, but have the output be in a separate block.